docs/howtoguides/enable_livepatch.md

```diff
@@ -22,13 +42,47 @@ Installing canonical-livepatch snap
 Canonical livepatch enabled.
 ```
 
-To check the status of Livepatch once it has been installed use the following
-command:
+## Check Livepatch status after installation
+To check the status of the Livepatch client once it has been installed use the
+following command:
 
 ```console
 $ sudo canonical-livepatch status
 ```
 
-```{seealso}
-For more information, see: https://ubuntu.com/security/livepatch
+### Unsupported kernels
+
+Although you can enable Livepatch on an unsupported kernel, since patches are
+kernel-specific, you will not receive any updates from Livepatch if your kernel
+is not supported. The `canonical-livepatch status` command will warn you if
+your kernel is unsupported (output truncated for brevity):
+
+```console
+...
+server check-in: succeeded
+kernel state: ✗ kernel not supported by Canonical 
+patch state: ✓ no livepatches needed for this kernel yet
+...
 ```
+
+You can also check [the support matrix](https://ubuntu.com/security/livepatch/docs/kernels)
+to see if your kernel is supported by Livepatch. To find out more, refer to
+this explanation of
+[how Livepatch works](https://ubuntu.com/security/livepatch/docs/livepatch/explanation/howitworks).
+
+## How to disable Livepatch
+
+Enabling Livepatch installs the Livepatch client as snap package, and there are
+a few possible ways to disable it. To stop the service, follow this short guide
+on [how to disable the client](https://ubuntu.com/security/livepatch/docs/livepatch/how-to/disable)
+from the Livepatch documentation.
```

Comment:
The linked article is more "How to configure livepatch to not automatically apply patches" and isn't analogous to the "How to disable" sections of our other services.

We can reference that document, but our "How to disable" should essentially be `pro disable livepatch` with an optional second step of `snap remove canonical-livepatch`.

Reply:
Oops, yes, that was me copying the wrong link - sorry!

Reply:
Good note, I've reworked the paragraph and hopefully it's better!